test/integration/comments_creation_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+class CommentsCreationTest < ActionDispatch::IntegrationTest
+  
+  attr_accessor :user#, :post
+
+  def setup
+    @user = users(:michael)
+    @post = posts(:post1)
+  end
+
+  test "should display the comments under the article commented with valid information login" do
```

Comment:
"when we comment an article, should display the new comment", ça me parait mieux